examples/03_connectivity/plot_compare_decomposition.py

```diff
@@ -129,6 +129,23 @@
     plot_stat_map(cur_img, display_mode="z", title="Comp %d" % i,
                   cut_coords=1, colorbar=False)
 
+###############################################################################
+# Estimate explained variance per component and plot using matplotlib
+#
+# The fitted object `dictlearning` can be used to calculate the score per component
+scores = dict_learning.score(func_filenames, per_component=False)
```

Comment:
```suggestion
scores = dict_learning.score(func_filenames, per_component=True)
```
(?)

Reply:
>(?)

Yes, that should be the right one. But, I am checking something. Will get back to ```True```